### DISCUSSION TITLE
Nuxt 3 SEO optimization

### DISCUSSION THREAD

**Discussion by dymitr8** [nuxt/nuxt#24171] [Questions]
Hey everyone

I've built a small web app, and it's essentially a landing page with smooth scroll navigation via header links and Nitro server handling Nodemailer + Firebase functionality.

Right now, I'm in the polishing stage, and I'm curious about SEO configuration. I've discovered useHead and useSeoMeta, which are quite useful. I've also read a few related articles and ran into some confusion when you're not sure what you actually need for your specific requirements.

What should I do and what shouldn't I do in this case? Any thoughts?

**Comment by TechAkayy** [comment]
I would highly recommend the https://nuxtseo.com module.
